Mathlib/SetTheory/Cardinal/Arithmetic.lean

```diff
@@ -642,10 +229,7 @@ theorem mul_eq_left_iff {a b : Cardinal} : a * b = a ↔ max ℵ₀ b ≤ a ∧
     · rw [mul_eq_max_of_aleph0_le_left ha hb, max_eq_left hab]
     all_goals simp
 
-end mul
-
 /-! ### Properties of `add` -/
-section add
```

Comment:
What's the purpose of removing all the `section`s throughout this file? They're often valuable, for instance navigating a file using Explorer view in VSCode.

Reply:
I felt like they were made redundant by the section headers. I don't think the pattern of
```
/-! ### Foo -/
section foo

end foo
```
is all that common (and imo if you're not introducing variables or anything it's just more bloat).

I'll introduce them back if you want though.

Reply:
> I don't think the pattern of
> 
> ```
> /-! ### Foo -/
> section foo
> 
> end foo
> ```
> 
> is all that common

I actually think this is *the* pattern?

Reply:
Well, I've added the sections back.